v2/client/http_receiver.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func NewHTTPReceiveHandler(ctx context.Context, p *thttp.Protocol, fn interface{}) (*EventReceiver, error) {
-	invoker, err := newReceiveInvoker(fn)
+	invoker, err := newReceiveInvoker(fn, noopObservabilityService{})
```

Comment:
We should get ahead of this and add a observability.Get() that defaults to `noopObservabilityService` here and everywhere else. We can do this as a follow up but it should happen before we cut

Reply:
sure